src/components/App/index.js

```diff
@@ -98,28 +77,64 @@ export const App = ({ options }) => {
       ? switchToHall()
       : switchToRules();
   }
-
+  
+  let nextQuestion = {}; 
+  const arrGameOverResuls = [];
+  let currentPlayerAnswer = "";
+  let quiz = {};
   //get data from API based on active game mode
   const apiDataFetcher = new ApiDataFetcher(options.swApiBaseUrl);
   playTheGame.addEventListener('click', () => play(config.selectedGame));
 
   async function play(gameMode) {
-    const quiz = new GameEngine(gameMode, apiDataFetcher);
+    quiz = new GameEngine(gameMode, apiDataFetcher);
 
     await quiz.fetchAllQuestionsForMode(gameMode);
 
     const gameView = new GameView();
-    const nextQuestion = quiz.generateNextQuestion();
+    nextQuestion = quiz.generateNextQuestion();
     gameView.displayQuestion(nextQuestion);
+ 
   }
+  
+
+  inGameMode.addEventListener('click', (event) => {
+    const isButton = event.target.nodeName === 'BUTTON';   
+    currentPlayerAnswer = event.target.innerText;
+      if (isButton) {
+        const computerMind = new ComputerMind();
+        const setGameLevel  = computerMind.setGameLevel(nextQuestion);
+        const playersSelection = computerMind.RandomComputerAnswer(setGameLevel, nextQuestion)
+        const result = gameOverView(playersSelection);
+        console.log(result);
+        const gameView = new GameView();
+        nextQuestion = quiz.generateNextQuestion();
+        gameView.displayQuestion(nextQuestion);
+   
+      }   
+   
+       
+    })
+
+    function gameOverView(selectedAnswers){
+      let keys = Object.keys(selectedAnswers)    
```

Comment:
Referencje do obiektów( w tym też tablic) się nie zmieniają więc powinniśmy je trzymać pod `const` by sobie przypadkiem nie nadpisać.

Swoją drogą to jeżeli dobrze rozumiem tu się zawsze ta sama nazwa wyciągnie, więc można ją chyba wpisać na sztywno, lub lepiej uprościć obiekt zwracany w GameMind

Reply:
poprawione na const